website/server/controllers/api-v3/user.js

```diff
@@ -28,7 +28,29 @@ let api = {};
  * @apiGroup User
  *
  * @apiSuccess {Object} data The user object
- */
+ *
+ * @apiDescription The user profile contains data related to the authenticated user including (but not limited to);
+ * Achievements
+ * Authentications (including types and timestamps)
```

Comment:
My personal feeling is that it would be enough to display something like this:

```
{
  "success": true,
  "data": {
    "_id": "11be077e-4a54-4d3a-96b6-8f14c101aeff",
    [All applicable user data goes here. Fetch data from your own account as an example. See website/server/models/user/* for details.]
  }
}
```

A partial list could be misleading, and the full list is too large as you noted, and is updated too often as paglias mentioned. Even a partial list is likely to become outdated fairly quickly and we'd almost always forget to update it. Anyone who wants to know what's in the data can test it  easily enough with their own account.

However if other admins disagree, I'm fine with whatever they suggest. :)